components/HeroSection.vue

```diff
@@ -129,8 +196,12 @@ section {
 }
 
 .cta-button {
```

Comment:
Curious Q: Can we call it as just `CTA` instead of cta-button?

cc: @ajithranka

Reply:
I think the distinction was used because we have both `cta-button` and `cta-icon`.